packages/strapi-plugin-graphql/hooks/graphql/index.js

```diff
@@ -95,6 +95,25 @@ module.exports = strapi => {
         schemaDef.resolvers = resolvers;
       }
 
+      // TODO: Remove these deprecated options in favor of `apolloServerConfig` in the next major version
+      const deprecatedApolloServerConfig = {
+        tracing: _.get(strapi.plugins.graphql, 'config.tracing', false),
+        introspection: _.get(strapi.plugins.graphql, 'config.introspection', true),
+        engine: _.get(strapi.plugins.graphql, 'config.engine', false),
+      };
+
+      if (
+        strapi.plugins.graphql.config.tracing ||
```

Comment:
you will have to check with a has instead or the warning won't display if the vars arf falsy

Reply:
Ops! You'r right! Fixed!